cloudflarekv/cloudflarekv_test.go

```diff
@@ -0,0 +1,118 @@
+package cloudflarekv
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var testStore *Storage
```

Comment:
Create a store per test, having global stores is something we have to remove from all drivers.

Reply:
By creating a store per test, you can all run the tests in parallel with `t.Parallel()`